src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -142,23 +156,81 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
+    // Issue commands to the speechSynthesis interface with deferral (1 ms timeout);
+    // this makes the wrapper behave better when issuing commands, especially
+    // play and pause
+    fluid.textToSpeech.deferredSpeechSynthesisControl = function (control, args) {
+        setTimeout(function () {
+            speechSynthesis[control](args);
+        }, 1);
+    };
+
+    // Throttle implementation adapted from underscore.js 1.8.3; see
+    // file header for license details
+    // Returns a version of a function that will only be called max once
+    // every "wait" MS
+    fluid.textToSpeech.throttle = function (func, wait, options) {
+        var timeout, context, args, result;
+            var previous = 0;
+            if (!options) options = {};
+
+            var later = function() {
+              previous = options.leading === false ? 0 : new Date().getTime();
+              timeout = null;
+              result = func.apply(context, args);
+              if (!timeout) context = args = null;
+            };
+
+            var throttled = function() {
+              var now = new Date().getTime();
+              if (!previous && options.leading === false) previous = now;
+              var remaining = wait - (now - previous);
+              context = this;
+              args = arguments;
+              if (remaining <= 0 || remaining > wait) {
+                if (timeout) {
+                  clearTimeout(timeout);
+                  timeout = null;
+                }
+                previous = now;
+
+                result = func.apply(context, args);
+                if (!timeout) context = args = null;
+              } else if (!timeout && options.trailing !== false) {
+                timeout = setTimeout(later, remaining);
+              }
+              return result;
+            };
+
+            throttled.cancel = function() {
+              clearTimeout(timeout);
+              previous = 0;
+              timeout = context = args = null;
+            };
+            return throttled;
+    };
+
+    // Throttled version of deferred speech synthesis control
+    fluid.textToSpeech.throttleControl = fluid.textToSpeech.throttle(fluid.textToSpeech.deferredSpeechSynthesisControl, 100, {leading: false});
+
     fluid.textToSpeech.speak = function (that, speaking) {
         that.events[speaking ? "onStart" : "onStop"].fire();
     };
 
-    fluid.textToSpeech.pause = function (that, paused) {
-        if (paused) {
-            that.events.onPause.fire();
-        } else if (that.model.speaking) {
-            that.events.onResume.fire();
+    fluid.textToSpeech.requestControl = function (that, control, change) {
+        // If there's a control request (value change to true), clear and
+        // execute it
+        if(change.value) {
```

Comment:
What fails (the sequel) is actually my ability to install my precommit hooks on my laptop as well as my work desktop to make sure I can't commit unlinted code.